docs/analyzer/analyzer_engine_provider.md

```diff
@@ -174,3 +174,81 @@ nlp_configuration:
 ```
 
 In this example, the `SpacyRecognizer` is disabled, and the `CreditCardRecognizer` is enabled, resulting in only the `CREDIT_CARD` PII entity to be returned if detected.
+
+### Adding context words in YAML recognizers
+
+Recognizers defined in YAML can also include a `context` field.
+When used with `AnalyzerEngine` and a context enhancer, these words boost the score if they appear near the detected entity.
+
+Example:
+
+```yaml
+recognizers:
+  - name: "Date of Birth Recognizer"
+    supported_entity: "DATE_TIME"
+    supported_language: "en"
+    patterns:
+      - name: "DOB without slashes"
+        regex: "((19|20)\\d{2}(0[1-9]|1[0-2])(0[1-9]|[12]\\d|3[01]))"
+        score: 0.8
+    context:
+      - DOB
+
+### Reading pattern recognizers from YAML
+
+#### 1. YAML Example
+
+recognizers:
+  - name: "Date of Birth Recognizer"
+    supported_entity: "DATE_TIME"
+    supported_language: "en"
+    patterns:
+      - name: "DOB without slashes"
+        regex: "((19|20)\\d{2}(0[1-9]|1[0-2])(0[1-9]|[12]\\d|3[01]))"
+        score: 0.8
+    context:
+      - DOB
+
+python <<EOF
+from presidio_analyzer import AnalyzerEngine, RecognizerRegistry
+from presidio_analyzer.nlp_engine import NlpEngineProvider
+from presidio_analyzer.context_aware_enhancers.lemma_context_aware_enhancer import LemmaContextAwareEnhancer
+
+# Configure NLP engine
+configuration = {
+    "nlp_engine_name": "spacy",
+    "models": [{"lang_code": "en", "model_name": "en_core_web_sm"}],
+}
+provider = NlpEngineProvider(nlp_configuration=configuration)
+nlp_engine = provider.create_engine()
+
+# Load recognizer from YAML
+registry = RecognizerRegistry()
+registry.add_recognizers_from_yaml("dob_recognizer.yml")
```

Comment:
This assumes that the dob_recognizer is already saved to disk. Could you please add this step before calling this method?